akka-docs/rst/scala/code/docs/http/scaladsl/HttpServerExampleSpec.scala

```diff
@@ -254,16 +249,14 @@ class HttpServerExampleSpec extends WordSpec with Matchers {
       val route =
         path("hello") {
           get {
-            complete {
-              <h1>Say hello to akka-http</h1>
-            }
+            complete("<h1>Say hello to akka-http</h1>")
           }
         }
 
       val bindingFuture = Http().bindAndHandle(route, "localhost", 8080)
 
       println(s"Server online at http://localhost:8080/\nPress RETURN to stop...")
-      Console.readLine() // for the future transformations
+      StdIn.readLine() // for the future transformations
```

Comment:
The comment is not really helpful. (I know it has been here before). Lets change it to `wait for user input` or `delay future transformations until user input`. WDYT?